shell/platform/android/io/flutter/Build.java

```diff
@@ -0,0 +1,27 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+package io.flutter;
+
+/** A replacement of utilities from android.os.Build. */
+public class Build {
+  /** For use in place of the Android VERSION CODES class. */
```

Comment:
Rewrite the comment so if someone searched for the official class they would find this:

/** For use in place of Build.VERSION_CODES class */

Reply:
I can do that, but I'll have to special case the forbidden imports script to allow it in this file.

Reply:
Can the linter detect that text is contained inside of a comment?